packages/ui-components/src/components/UITranslationInput/defaults.ts

```diff
@@ -1,16 +1,17 @@
 import type { TranslationInputStrings } from './UITranslationButton.types';
 
 export const defaultTranslationInputStrings: TranslationInputStrings = {
-    acceptButtonLabel: 'Apply',
+    acceptButtonLabel: 'Substitute',
```

Comment:
Only not sure about this label `Substitute` - I think it is not always `Substitute`